integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/file/inbound/transport/test/InboundEndpointContentTypePlainTest.java

```diff
@@ -44,8 +44,7 @@
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
 
-	    pathToFtpDir = FrameworkPathUtil.getSystemResourceLocation() + File.separator + "artifacts" + File.separator
-	                   + "ESB" + File.separator + "synapseconfig" + File.separator + "vfsTransport";
+	    pathToFtpDir = getESBResourceLocation() + File.separator + "synapseconfig" + File.separator + "vfsTransport";
```

Comment:
Make sure this is not exceeding the length 120

Reply:
It is not. It's 118. :)